github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/config.groovy

```diff
@@ -39,8 +39,10 @@ f.block {
             )
         }
 
-        f.optionalProperty(title: "User Restriction", field: "userRestriction")
-   
-        f.optionalProperty(title: "Branch Restriction", field: "branchRestriction")
+        f.advanced() {
```

Comment:
Better prefix with "Experimental", would be cool to have such marked automatically in UI based on class annotation.